context/build.gradle

```diff
@@ -0,0 +1,16 @@
+plugins {
+    id "be.insaneprogramming.gradle.animalsniffer" version "1.4.0"
+}
+
+description = 'gRPC: Context'
+
+dependencies {
+    compile libraries.guava,
```

Comment:
yeah looking at the code it would be trivial to move both of these to test deps. Things like context are best as dependency-free libraries.